app/factories/LobbyFactory.js

```diff
@@ -6,7 +6,7 @@ function LobbyFactory(){
   return {
     createLobby: createLobby
   };
-  function createLobby(options, port){
+  function createLobby(options, port, path, GameServerPort){
```

Comment:
For extra clarity, call the `path` argument `configPath`, assuming it is indeed that